src/Lesson1/Lesson1.java

```diff
@@ -0,0 +1,48 @@
+package Lesson1;
+
+public class Lesson1 {
+    public static void main(String[] args) {
+        printThreeWords();
+        checkSumSign();
+        printColor();
+        compareNumbers();
+    }
+    public static void printThreeWords() {
+        System.out.println("Orange");
+        System.out.println("Banana");
+        System.out.println("Apple");
+    }
+    public static void checkSumSign(){
+        int a = -5;
+        int b = 14;
+        int c = a + b;
+        if (c >= 0) {
+            System.out.println("Сумма положительная");
+        }
+        if (c < 0) {
+            System.out.println("Сумма отрицательная");
+        }
+    }
+    public static void printColor() {
+        int value = 155;
+        if (value <= 0) {
+            System.out.println("Красный");
+        }
+        if (value > 0 && value <= 100) {
+            System.out.println("Желтый");
+        }
+        if (value > 100) {
+            System.out.println("Зеленый");
+        }
```

Comment:
```suggestion
        if (value <= 0) {
            System.out.println("Красный");
        } else if (value > 0 && value <= 100) {
            System.out.println("Желтый");
        } else if (value > 100) {
            System.out.println("Зеленый");
        }
```
лучше сделать в рамках одного оператора, чтобы при первом положительном условии исключить проверки всех остальных, поскольку они взаимоисключающие